### ISSUE TITLE
Warnings in Jest

### ISSUE THREAD

**Issue by eckdanny-osi** [ca-cwds/case-management#143]
jest test runs as of 660429f raise the following warning:

```
  console.error node_modules/fbjs/lib/warning.js:33
    Warning: validateDOMNesting(...): Text nodes cannot appear as a child of <tbody>.
        in tbody (created by Table)
        in table (created by Table)
        in div (created by Table)
        in Table (created by Ethnicity)
        in div (created by Ethnicity)
        in div (created by Ethnicity)
        in div (created by Cards)
        in div (created by Cards)
        in div (created by Cards)
        in Cards (created by Ethnicity)
        in Ethnicity (created by ClientIdPage)
        in div (created by ClientIdPage)
        in div (created by ClientIdPage)
        in div (created by ClientIdPage)
        in div (created by ClientIdPage)
        in ClientIdPage (created by WrapperComponent)
        in WrapperComponent
```

Bisecting indicates 362b0a0 (Ethnicity card (#121)) to be the first commit in which the related issue begins to appear.

**Comment by eckdanny-osi** [comment]
related to #160

**Comment by RamuKammagani-osi** [comment]
Fixed in the pull request  #169! Are we good to close this?
